cypress/e2e/data-import/importing-marc-bib-files/disallow-editing-of-instance-records-that-have-underlying-srs-marc-records.cy.js

```diff
@@ -0,0 +1,89 @@
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import Users from '../../../support/fragments/users/users';
+import TopMenu from '../../../support/fragments/topMenu';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import JsonScreenView from '../../../support/fragments/data_import/logs/jsonScreenView';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import InstanceRecordView from '../../../support/fragments/inventory/instanceRecordView';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import FileDetails from '../../../support/fragments/data_import/logs/fileDetails';
+import InstanceRecordEdit from '../../../support/fragments/inventory/instanceRecordEdit';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+
+describe('Data Import', () => {
+  describe('Importing MARC Bib files', () => {
+    let user;
+    let instanceHrid;
+    const filePath = 'marcFileForC2361.mrc';
+    const marcFileName = `C2361 autotestFileName ${getRandomPostfix()}`;
+    const jobProfileToRun = 'Default - Create instance and SRS MARC Bib';
+    const title = "101 things I wish I'd known when I started";
+
+    before('login', () => {
+      cy.createTempUser([
+        Permissions.moduleDataImportEnabled.gui,
+        Permissions.settingsDataImportEnabled.gui,
+        Permissions.inventoryAll.gui,
```

Comment:
The user should has only 'Inventory: All permissions' permission

Reply:
Fixed